pkg/gen/typegen.go

```diff
@@ -916,6 +916,74 @@ func createGroups(definitionsJSON map[string]interface{}, opts groupOpts) []*Gro
 		}).
 		ToSlice(&definitions)
 
+	// Compute aliases for Kinds. Many k8s resources have multiple GVs, so create a map from Kind -> GV string.
+	// For Kinds with more than one GV, create aliases in the SDKs.
+	type SimpleKind struct {
+		kind       string
+		apiVersion string
+	}
+	aliases := map[string][]interface{}{}
+	linq.From(definitions).
```

Comment:
I just want to say, whoever wrote linq for go is both a hero and also this is extremely cursed code thanks to the lack of generics

@ahmetb: thanks, I hate it! ;)